frontend/src/concepts/pipelines/content/createRun/CloneRunPage.tsx

```diff
@@ -10,13 +10,13 @@ const CloneRunPage: PipelineCoreDetailsPageComponent = ({ breadcrumbPath, contex
 
```

Comment:
@Gkrumbach07 , that was not mentioned in the issue , but if everyone agrees with it , I will make the necessary changes

Reply:
This is a tough point of contention usually. On one hand, it makes sense to keep code related to the text we show the user -- makes it easier to find it by pure text string looking up. On the other hand, we have technology to help support reversing from text on screen. For one, text search of the button text and working backwards, or using the React Dev Tools as well.

This is the same concern around Notebook vs Workbench in the UI. I think it is easier to name things on technical use and when they are purely visual representations (ie: `CloneButton`) that it might be better to rename them.

In this case, it's technically about the "cloning" of a run and not really tied to the naming of it. (if you had renamed it, it would have been okay too -- not the end of the world).